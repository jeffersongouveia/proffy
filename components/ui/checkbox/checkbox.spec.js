import React from "react";
import Checkbox from "./index.js";
import ReactDOM from "react-dom";
import Enzyme, { mount } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });

describe("Checkbox specifications", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = mount(<Checkbox />);
  });

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
    wrapper.unmount();
  });
});
