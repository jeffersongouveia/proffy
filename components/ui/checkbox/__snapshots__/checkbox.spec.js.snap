// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox specifications should match snapshot 1`] = `
<Checkbox>
  <div>
    <Checkbox
      indeterminate={false}
    >
      <label
        className="ant-checkbox-wrapper"
      >
        <Checkbox
          className=""
          defaultChecked={false}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          prefixCls="ant-checkbox"
          style={Object {}}
          type="checkbox"
        >
          <span
            className="ant-checkbox"
            style={Object {}}
          >
            <input
              checked={false}
              className="ant-checkbox-input"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              type="checkbox"
            />
            <span
              className="ant-checkbox-inner"
            />
          </span>
        </Checkbox>
      </label>
    </Checkbox>
  </div>
</Checkbox>
`;
